crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -5725,7 +5770,7 @@ mod tests {
         // and the inferred type will just be `BaseExceptionGroup` --Alex
         //
```

Comment:
This comment (including the two previous lines GH won't let me comment on) can be removed.